### DISCUSSION TITLE
How to removing google fonts & google static?

### DISCUSSION THREAD

**Discussion by git-gnulf** [givanz/Vvveb#215] [Q&A]
Hello everyone,

is it possible to remove all external Google fonts and Google Static resources? And if so, how? 

One of the reasons for this is that I need to prevent external resource calls. Especially with regard to the GDPR.

Can this also be done via the interface or do I have to intervene manually in the themes?

Do any of you have any tips?
That would be great.

Thank you very much.

**Comment by givanz** [comment]
Hi

Default themes like landing and default blog don't include google fonts, they load only local fonts.

To disable google fonts in page builder and remove google fonts `<link>`s from themes like 

```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
```

you can install this plugin https://plugins.vvveb.com/product/remove-google-fonts 

If any theme is loading Google fonts from css files they must be removed manually.
